tcrdb/resources/schemas/dbscripts/sqlserver/tcrdb-15.50-15.51.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE tcrdb.stims ADD tissue varchar(4000);
```

Comment:
Not sure if you care about supporting extended characters, but may want this to be NVARCHAR instead.

Reply:
yeah, i know.  There are just enough tables that a while ago I decided that I should either convert everything or continue as-is.  At some point perhaps I'll work out some java update code that systematically converts everything, but it hasnt been a problem so far.